src/util/saveArticles.ts

```diff
@@ -28,6 +28,11 @@ type FileStore = RedisKvs<{
     width?: number;
 }>;
 
+function isImageUrl(url: string) {
+    url.toLowerCase();
+    return (url.substr(url.length-4) === 'jpeg' || url.substr(url.length-3) === 'jpg' || url.substr(url.length-3) === 'png' || url.substr(url.length-3) === 'gif')
+}
+
```

Comment:
No:
* Same name than an other method of the same software which does something similar but not identical
* Use regex for this kind of work

Just remove this, more infos inthe next comment